### PR TITLE
a few changes by timmy

### PR DESCRIPTION
**Pull request by timmy471**

### CODE REVIEW

**Review by MyMedicalBank — CHANGES_REQUESTED** [review]
This change seems like a joke. Can you kindly look through an apply a reasonable code update.

File: pagecarton/core/application/modules/Application/Article/ShowAll.php [hunk 1]
```diff
@@ -222,7 +222,7 @@ public function showMessage()
 	//	$this->_parameter['markup_template'] = null;
 		if( empty( $_GET['pc_post_list_id'] ) )
 		{
-			$message = array_pop( $this->_badnews ) ? : 'Posts will be displayed here when they become available.';
+			$message = array_pop( $this->_badnews ) ? : 'Posts here when they become available.';
```
Comment (MyMedicalBank) [inline pagecarton/core/application/modules/Application/Article/ShowAll.php:225]: What is happening here?

File: pagecarton/core/application/documents/layout/bootstrapbasic/index.html [hunk 2]
```diff
@@ -40,7 +40,8 @@
                     <span class="icon-bar"></span>
                 </button>
                 <a class="navbar-brand" href="#">
-                    <img src="http://placehold.it/150x50&text=Logo" alt="">
+                    <img src="https://www.google.com/imgres?imgurl=https%3A%2F%2Fmedia.gettyimages.com%2Fphotos%2Fgold-letter-t-picture-id186850987%3Fs%3D612x612&imgrefurl=https%3A%2F%2Fwww.gettyimages.com%2Fphotos%2Fletter-t&tbnid=WUXfqUKDg0pGJM&vet=12ahUKEwibsY_n_
```
Comment (MyMedicalBank) [inline pagecarton/core/application/documents/layout/bootstrapbasic/index.html]: What is happening here?

File: pagecarton/core/application/documents/css/pagecarton.css [hunk 3]
```diff
@@ -53,7 +53,7 @@ Basic Style
     content: '';
     height: 16px;
     width: 16px;
-    background: url(../img/cd-custom-separator.svg) no-repeat center center;
+    background: url(../img/abstract.jpg) no-repeat center center;
```
Comment (MyMedicalBank) [inline pagecarton/core/application/documents/css/pagecarton.css]: Where is this file u just added?

**Review by ayoolafalola — COMMENTED** [review]
I can't seem to wrap my head around the  changes you are trying to effect.

**Pull request closed without merging**